corehq/apps/app_manager/ui_translations/ui_translations.py

```diff
@@ -68,10 +69,14 @@ def _get_params_text(params):
         return ", ".join(["${{{}}}".format(num) for num in numbers])
 
     for row in translations:
+        if row["property"] == "":
+            warnings.append(_("Property '' is empty, "
+                            "We did not add it to the translations"))
+            continue
         if row["property"] not in commcare_ui_strings:
             # Add a warning for  unknown properties, but still add them to the translation dict
-            warnings.append("Property '" + row["property"] + "' is not a known CommCare UI string, "
-                            "but we added it anyway.")
+            warnings.append(_("Property {prop} is not a known CommCare UI string, "
+                            "but we added it anyway.").format(prop=row["property"]))
```

Comment:
nit: Would it make sense to put the property name in quotes? Imagine a property named 'name', which would result in an ambiguous and possibly confusing message like "Property name is not a known CommCare UI string..."
```suggestion
            warnings.append(_("Property '{prop}' is not a known CommCare UI string, "
                            "but we added it anyway.").format(prop=row["property"]))
```